ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1077,3 +1077,195 @@
                     default-excludes: true
                     fingerprint: false
                     only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
```

Comment:
May I know if all new items are dual-stack jobs? I can't find any ipv6-only jobs in lab or cloud yamls.

Reply:
**jobs with "-ipv6-" name are "ipv6-only"  jobs. e.g**
```
antrea-kind-aws-ipv6-conformance-for-pull-request
antrea-kind-aws-ipv6-e2e-for-pull-request
antrea-kind-aws-ipv6-networkpolicy-for-pull-request
```
**jobs with "-ds-"  name are "dual-stack" jobs e.g**
```
antrea-kind-aws-ds-conformance-for-pull-request
antrea-kind-aws-ds-e2e-for-pull-request
antrea-kind-aws-ds-networkpolicy-for-pull-request
```
I thought we don't have to specify "-only" but if you want me to add ipv6-only instead of ipv6 in the jobs so I can do that?

Reply:
@KMAnju-2021 , we shld add "ipv6-only"

Reply:
Yes，we should follow the same naming convention as the existing IPv6 jobs https://github.com/antrea-io/antrea/blob/7bdf76b1edff8325ff63393b70633d92c17bb27d/ci/jenkins/jobs/projects-lab.yaml#L135, using `ipv6-ds` and `ipv6-only`. @KMAnju-2021